posts/Ippon-USA-offers-a-helping-hand-to-those-impacted-by-natural-disasters.md

```diff
@@ -0,0 +1,32 @@
+Background:
+
+On September 7, 2017, one of the most lethal earthquakes in Mexican history hit close to the southern state of Chiapas, with a devastating magnitude of 8.2. Just 12 days later and a few hundred miles away, a 7.1 magnitude quake rocked central Mexico, toppling buildings, breaking gas mains, knocking out electricity and sparking fires across the city and other towns in central Mexico.
+
+
+
+More than a year after the devastating Earthquakes hit, many schools in Mexico remained unsafe for children to learn in, which forced many students to study in dangerous temporary learning centres. For this reason, we are continuing our work in Mexico within the Mexican state of Oaxaca.
+The Oaxaca program began in January 2019, focusing on assisting the Celso Muñoz Primary School within the Oaxaca state, where our work will be centered around rebuilding 7 classrooms, retrofitting 4 classrooms and rebuilding the office and library to ensure that students have a safe and disaster-resilient place to learn. This school will allow 170 students, between the ages of 6 and 12 to return to school and receive the education they need for a brighter future. We look forward to working with this diverse and culturally-rich community to rebuild a school that will ensure they build a safer future for their students that will allow their diversity of culture to thrive for many years to come.
+My Time on Project:
+15 hours after setting off from my apartment in NYC and taking nearly every mode of transport available, I arrived at the HQ of the volunteer program in Oaxaca, Mexico. The so-called base is located in small neighbourhood La Santita, a barrio of Salinas Del Marquez. This town is separated by a massive harbour from the more well known Salina Cruz, famous for the oil refinery. 
+The next morning, after waking up early, the whole group of volunteers headed to the work site at 6.40 am. Currently, the group consist of approximately 40 people of all different ages from all over the world. All in purple, work boots on, sunscreen, water bottle and with a positive attitude: off to ‘work’. The volunteers make 8 long hour days, 6 days a week, in the extreme heat that marks this coastal area. Every now and then, they are blessed with a breeze. The region is known for high winds, but this year they are very absent, unfortunately. 
+On site, the group of volunteers works closely together with 15 masons, local labourers hired by the NGO. This comes with some difficulties from time to time because of the language barriers, just one of them speaks English and majority of the volunteers do not speak Spanish. Mostly, they work it out with sign language and the couple of words they know and it turns into a hilarious situation. Once a week, after work, the masons and volunteers will have ‘Spanglish’ classes, to learn the basics of the other language. 
```

Comment:
```suggestion
On-site, the group of volunteers works closely together with 15 masons, local laborers hired by the NGO. There are some difficulties from time to time because of the language barriers; just one of them speaks English, and the majority of the volunteers do not speak Spanish. Mostly, they work it out with sign language and the couple of words they know, and it turns into a hilarious situation. Once a week, after work, the masons and volunteers will have ‘Spanglish’ classes, to learn the basics of the other language. 
```